examples/make_assets/sankey.py

```diff
@@ -8,15 +8,16 @@
 
 # %% Sankey diagram of random integers
 cols = ["col_a", "col_b"]
-df_rand_ints = pd.DataFrame(np.random.randint(1, 6, size=(100, 2)), columns=cols)
+np_rng = np.random.default_rng(seed=0)
+df_rand_ints = pd.DataFrame(np_rng.randint(1, 6, size=(100, 2)), columns=cols)
```

Comment:
I haven't got time to look through everything yet (perhaps would do tomorrow if you want), this should be `np_rng.integers()`.